### DISCUSSION TITLE
How to make button group go from vertical to row

### DISCUSSION THREAD

**Discussion by UmbrellaCrow612** [saadeghi/daisyui#1084] [Q&A] [answered]
Hi

When i have a mobile view i place the button group in a vertical position with the classname but there isnt a way for me a md: breakpoint to make it into a row how would i do this ?

**Comment by saadeghi** [comment, marked as answer]
Please update to version `2.27.0`  
`btn-group-horizontal` and `btn-group-vertical` class names are now responsive ([docs](https://daisyui.com/components/button-group/))  
https://play.tailwindcss.com/F9sWxf0lDM
